source/euclid.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Нахождение НОД для двух чисел
+ *
+ * @param {number} x - число, для которого ищется НОД
+ * @param {number} y - число, для которого ищется НОД
+ * @returns {number} x - НОД для двух чисел
+ */
+const gcd = (x, y) => y ? gcd(y, x % y) : x
+
+/**
+ * Нахождение НОД для любого количества чисел
+ *
+ * @param {...numbers} numbers - любое количество чисел
```

Comment:
```suggestion
 * @param {...number} numbers - любое количество чисел
```
это названия типа данных `number` - как `string` или `boolean`